src/llama_stack_client/lib/agents/agent.py

```diff
@@ -353,7 +350,7 @@ def __init__(
         :param instructions: The instructions for the agent.
         :param tools: A list of tools for the agent. Values can be one of the following:
             - dict representing a toolgroup/tool with arguments: e.g. {"name": "builtin::rag/knowledge_search", "args": {"vector_db_ids": [123]}}
-            - a python function decorated with @client_tool
+            - a python function with a docstring. See @client_tool for more details.
```

Comment:
not related to this PR, but I wonder if we could share these docstrings & remove need for duplicates for Agent & AsyncAgent somehow.